--- planet/vendor/feedparser.py	2008-08-11 20:15:10.000000000 +0800
+++ planet/vendor/feedparser.py	2009-02-15 03:08:14.000000000 +0800
@@ -2372,7 +2372,7 @@
       'option', 'p', 'pre', 'progress', 'q', 's', 'samp', 'section', 'select',
       'small', 'sound', 'source', 'spacer', 'span', 'strike', 'strong', 'sub',
       'sup', 'table', 'tbody', 'td', 'textarea', 'time', 'tfoot', 'th', 'thead',
-      'tr', 'tt', 'u', 'ul', 'var', 'video', 'noscript']
+      'tr', 'tt', 'u', 'ul', 'var', 'video', 'noscript', 'embed', 'object']
 
     acceptable_attributes = ['abbr', 'accept', 'accept-charset', 'accesskey',
       'action', 'align', 'alt', 'autoplay', 'autocomplete', 'autofocus', 'axis',
